src/store/modules/users.js

```diff
@@ -46,6 +67,9 @@ export default {
     allUsers(state) {
       return state.users;
     },
+    getUserById: (state) => (userId) => {
+      return state.users.find((user) => user.id === userId);
+    },
```

Comment:
This method won't fetch the correct data. The users array doesn't hold information about users' possessions. This needs to make a call to /resources/availability/{userId}